card/db/migrate_core_cards/20141204061304_watchers_to_following.rb

```diff
@@ -22,7 +22,7 @@ def up
     
       follower_hash.each do |user, items|
         if card=Card.fetch(user) and card.account
-          following = card.fetch :trait=>"following", :new=>{}
+          following = Card.fetch("#{card.name}+*following", :new=>{:type=>'pointer'})
```

Comment:
good points.

I keep thinking we might want to consider some new API for this.